pages/dao/[symbol]/proposal/components/instructions/TokenAccountSelect.tsx

```diff
@@ -0,0 +1,80 @@
+import { PublicKey } from '@solana/web3.js'
+
+import Select from '@components/inputs/Select'
+import { OwnedTokenAccountsInfo } from '@hooks/useGovernanceUnderlyingTokenAccounts'
+
+const TokenAccountSelect = ({
+  label,
+  value,
+  onChange,
+  error,
+  ownedTokenAccountsInfo,
+}: {
+  label: string
+  value?: string
+  onChange: (value: PublicKey) => void
+  error: string
+  ownedTokenAccountsInfo?: OwnedTokenAccountsInfo
+}) => {
+  if (!ownedTokenAccountsInfo) {
+    return <></>
+  }
+
+  const getAccountDisplay = (pubkey?: PublicKey) => {
+    if (!pubkey) return <div></div>
```

Comment:
all ownerTokenAccounts should have a public key ?

Reply:
getAccountDisplay return the data for a given pubkey. If there are no pubkey, then you can't return data. Which is the case when **_value_** is undefined (when nothing is selected)